src/app/(auth)/onboarding/layout.tsx

```diff
@@ -0,0 +1,19 @@
+'use client';
```

Comment:
Hmmm I know that this routing allows for us to have the onboarding `layout.tsx` inherit from the auth `layout.tsx`, but it feels like the routing may get a bit unwieldy if all the onboarding screens are in `src/app/(auth)/onboarding`, especially because onboarding isn't really part of auth. Thoughts?

Reply:
I think this is more of a question to discuss with all the devs, though it won't impact the user too much (will still be `/onboarding/...`). Personally I'm a fan of keeping it the way it is to keep consistency with Next.js' way of nesting layouts, since it looks pretty much the same. If we want to go for the `src/app/onboarding/` approach, we could, but I think to do so, we'll need a separate `AuthLayout` component that both uses, which doesn't feel as clean.